src/components/PathwayCourse/redux/api.js

```diff
@@ -24,3 +24,39 @@ export const getPathwaysCourse = (data) => {
     // headers: HeaderFactory(token),
   });
 };
+
+export const getUpcomingBatches = (data) => {
+  const { pathwayId, authToken } = data;
+  return axios({
+    method: METHODS.GET,
+    url: `${process.env.REACT_APP_MERAKI_URL}pathways/${pathwayId}/upcomingBatches`,
```

Comment:
In the server, we have `https://api.merakilearn.org/` so no big deal if we won't use `/` before the endpoint. In fact, we shouldn't use `/` anywhere before an endpoint.

Reply:
@Poonam-Singh-Bagh OK, I think it's more standard to have it without the `/` at the end (and this is what it seems to have previously been): https://github.com/navgurukul/bhanwari-devi/blob/c24fd8d099971eb0af0db3224d0b5750867e3405/deploy.sh#L8
But as long as it has the trailing `/` now, everything should be good.  And agreed, if it ends in `/`, then we don't want to have the extra leading `/` 's in absolute paths as we do now in some places because we don't want something like `https://api.merakilearn.org//courses`.

Reply:
What's more standard `https://api.merakilearn.org` or `https://api.merakilearn.org/`

Reply:
I think what it was originally, `https://api.merakilearn.org`.